prow/config/secrets_agent.go

```diff
@@ -14,9 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Implements an agent to read and reload the secrets.
 package config
 
+// Implements an agent to read and reload the secrets.
```

Comment:
This file should probably be in its own package..

Reply:
Under `prow/config` or else where?

Reply:
Not sure if I got you right, but here is an attampt: https://github.com/kubernetes/test-infra/commit/c7f7e9fc19297e78a4dd38fbf205e71f2068326c

Reply:
Thanks! That looks like a great PR in its own right!